lib/tls.js

```diff
@@ -39,10 +40,12 @@ exports.CLIENT_RENEG_WINDOW = 600;
 
 exports.SLAB_BUFFER_SIZE = 10 * 1024 * 1024;
 
-exports.DEFAULT_CIPHERS =
-    process.binding('constants').crypto.defaultCipherList;
-
-exports.DEFAULT_ECDH_CURVE = 'prime256v1';
+[ 'DEFAULT_CIPHERS', 'DEFAULT_ECDH_CURVE' ].forEach((key) => {
+  Object.defineProperty(exports, key, {
+    get: () => { return internalTLS[key]; },
+    set: (c) => { internalTLS[key] = c; }
+  });
```

Comment:
Defining accessors for otherwise simple data properties is kind of wasteful, just re-export the values from internal/tls.